### PR TITLE
Upgrade `ok_http` to `jnigen` 0.12.2

### PR DESCRIPTION
**Pull request by brianquinlan**
- Thanks for your contribution! Please replace this text with a description of what this PR is changing or adding and why, list any relevant issues, and review the contribution guidelines below.

---

- [ ] I’ve reviewed the contributor guide and applied the relevant portions to this PR.

<details>
  <summary>Contribution guidelines:</summary><br>

- See our [contributor guide](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md) for general expectations for PRs.
- Larger or significant changes should be discussed in an issue before creating a PR.
- Contributions to our repos should follow the [Dart style guide](https://dart.dev/guides/language/effective-dart) and use `dart format`.
- Most changes should add an entry to the changelog and may need to [rev the pubspec package version](https://github.com/dart-lang/sdk/blob/main/docs/External-Package-Maintenance.md#making-a-change).
- Changes to packages require [corresponding tests](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md#Testing).

Note that many Dart repos have a weekly cadence for reviewing PRs - please allow for some latency before initial review feedback.
</details>

### CODE REVIEW

**Comment by HosseinYousefi** [comment]
0.12.2 is published.

**Comment by brianquinlan** [comment]
FYI: @Anikate-De

File: pkgs/ok_http/pubspec.yaml [hunk 1]
```diff
@@ -13,13 +13,13 @@ dependencies:
     sdk: flutter
   http: ^1.2.1
   http_profile: ^0.1.0
-  jni: ^0.10.1
+  jni: ^0.12.0
```
Comment (HosseinYousefi) [inline pkgs/ok_http/pubspec.yaml:16]: Update this to 0.12.2 as well.

**Review by HosseinYousefi — APPROVED** [review]
LGTM with one comment.

**Pull request merged**